test/support/response.rb

```diff
@@ -0,0 +1,88 @@
+require 'cgi'
+
+ERROR_IDS = %w(400 401 404 500)
+
+def index_response(resource, count)
+  {
+    :total_count => count,
+    :has_more => false,
+    :object => 'list',
+    :data => FactoryGirl.build_list(resource.to_sym, count)
+  }.to_json
+end
+
+def resource_from_uri(uri)
+  uri.to_s.split('/').last.singularize.to_sym
+end
+
+# Make sure the stubbed URI is valid.
+def check_uri_for_api_key(uri)
+  unless uri.user
```

Comment:
Style/IfUnlessModifier: Favor modifier `unless` usage when having a single-line body. Another good alternative is the usage of control flow `&&`/`||`.